cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -316,3 +316,30 @@ def test_custom_button_expression_evm_obj(appliance, request, setup_obj, button_
             assert not custom_button_group.is_displayed
             setup_obj.add_tag(tag)
             assert button.text in custom_button_group.items
+
+
+@pytest.mark.manual
+def test_custom_button_open_url_evm_obj(setup_obj, button_group):
+    """ Test Open url functionality of custom button.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/2h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.11
+        casecomponent: CustomButton
+        tags: custom_button
+        testSteps:
+            1. Create ruby method for url functionality
+            2. Create custom button group with the Object type
+            3. Create a custom button with open_url option and respective method
+            4. Navigate to object Detail page
+            5. Execute custom button
+            6. Check new tab open or not with respective url
```

Comment:
I would expect to see `testSteps` and `expectedResults` for a manual test case here.